### PR TITLE
test: Update Toast codemod to transform before promote

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Because I was promoting toast and then converting to compound, the codemod was duplicating code. By switching the order, I convert toast to compound first and then I promote. 

## Release Category
Codemod

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

### CODE REVIEW

File: modules/codemod/lib/v9/spec/compoundToast.spec.ts [hunk 1]
```diff
@@ -135,4 +135,35 @@ describe('Toast', () => {
...
+        <Toast.Icon color="greenApple400" icon={checkIcon} />
+        <Toast.Body>
+          <Toast.Message>There was an error with your workbook.</Toast.Message>
+        </Toast.Body>
+      </Toast>
+
+    `;
+
+    expectTransform(input, expected);
+  });
```
Comment (mannycarrera4) [inline modules/codemod/lib/v9/spec/compoundToast.spec.ts:168]: This test verified that it should not duplicate order

File: modules/codemod/lib/v9/spec/compoundToast.spec.ts [hunk 2]
```diff
@@ -135,4 +135,35 @@ describe('Toast', () => {
 
     expectTransform(input, expected);
   });
+
+  it('should not restructure Toast usage if it is already compound', () => {
+    const input = stripIndent`
+      import {Toast} from '@workday/canvas-kit-labs-react/toast';
```
Comment (RayRedGoose) [inline modules/codemod/lib/v9/spec/compoundToast.spec.ts:141]: If you change order, make sure you change import path to `@workday/canvas-kit-labs-react/toast` for every other test

Comment (RayRedGoose) [reply modules/codemod/lib/v9/spec/compoundToast.spec.ts]: nvm

File: modules/codemod/lib/v9/index.ts [hunk 3]
```diff
@@ -9,8 +9,8 @@ const transform: Transform = (file, api, options) => {
   const fixes = [
     // add codemods here
     softDeprecateStack,
-    promoteToast,
     compoundToast,
+    promoteToast,
```
Comment (mannycarrera4) [inline modules/codemod/lib/v9/index.ts:13]: Promote toast after converting to compound

**Review by RayRedGoose — APPROVED** [review]

**Pull request merged**